src/integrationtests/java/com/aws/greengrass/integrationtests/certificateauthority/OfflineAuthenticationTest.java

```diff
@@ -277,6 +277,36 @@ void GIVEN_clientConnectsWhileOnline_WHEN_offline_THEN_clientCanConnect(Extensio
        assertNotNull(connectToCore(thingOne.get(), clientPem));
    }
 
+   @Test
+   void GIVEN_clientConnectsWhileOnline_WHEN_offlineAndTtlExpired_THEN_clientCanNotConnect(ExtensionContext context)
+           throws Exception {
+       ignoreExceptionOfType(context, NoSuchFileException.class);
+
+       // Given
+       network.goOnline();
+       Instant now = Instant.now();
+       mockInstant(now.toEpochMilli());
+
+       // Configure the things attached to the core
+       List<X509Certificate> clientCertificates = createClientCertificates(1);
+       String clientPem = CertificateHelper.toPem(clientCertificates.get(0));
+       Supplier<String> thingOne =  () -> "ThingOne";
+       iotAuthClientFake.attachCertificateToThing(thingOne.get(), clientPem);
+       iotAuthClientFake.attachThingToCore(thingOne);
+       iotAuthClientFake.activateCert(clientPem);
+
+       runNucleusWithConfig("config.yaml");
+       assertNotNull(connectToCore(thingOne.get(), clientPem));
+
+       // When
+       network.goOffline();
+       Instant twoMinutesLater = now.plusSeconds(2 * 60); // Default expiry is 1 minute
```

Comment:
This test could break if we ever update the default. We could refer to the static default value: `SecurityConfiguration.DEFAULT_CLIENT_DEVICE_TRUST_DURATION_MINUTES` instead of hard-coding.

Reply:
Yes, good call. I left a comment so hopefully it's easy to figure out in the future.

I'd recommend reading from config instead of relying on a constant in the source though